transport/http/handler.go

```diff
@@ -108,6 +108,11 @@ func (rw responseWriter) Write(s []byte) (int, error) {
 	return rw.w.Write(s)
 }
 
+// Options for the HTTP transport.
+func (responseWriter) Options() (o transport.Options) {
+	return o
```

Comment:
Using named params to init an empty `transport.Options` - is that an accepted idiom? cc @prashantv